virtcontainers/cc_proxy.go

```diff
@@ -5,28 +5,27 @@
 
 package virtcontainers
 
-import (
-	"os/exec"
-)
+import "os/exec"
```

Comment:
I don't think we're testing this any more in the CI? In fact, can we even drop the `cc_*` stuff now I wonder?

Reply:
Surely I agree. Created #694 to track it.